examples/dreambooth/train_dreambooth.py

```diff
@@ -150,7 +150,24 @@ def parse_args(input_args=None):
         default=None,
         help="Total number of training steps to perform.  If provided, overrides num_train_epochs.",
     )
-    parser.add_argument("--save_steps", type=int, default=500, help="Save checkpoint every X updates steps.")
+    parser.add_argument(
+        "--checkpointing_steps",
```

Comment:
I renamed the option to use the same as in https://github.com/huggingface/transformers/blob/799cea64ac1029d66e9e58f18bc6f47892270723/examples/pytorch/language-modeling/run_clm_no_trainer.py#L543.

Also, it doesn't include the name `save` to avoid confusion about what it means.